test/convolution-operator-tester.h

```diff
@@ -402,8 +401,11 @@ class ConvolutionOperatorTester {
           }
         }
       }
-      const int32_t accumulatorsMin = *std::min_element(accumulators.cbegin(), accumulators.cend());
-      const int32_t accumulatorsMax = *std::max_element(accumulators.cbegin(), accumulators.cend());
+      // Create dummy min/max for empty inputs.
+      // These are only used to compute scale and zero point,
+      // and real callers will just pull those values from the model.
+      const int32_t accumulatorsMin = accumulators.empty() ?   0 : *std::min_element(accumulators.cbegin(), accumulators.cend());
+      const int32_t accumulatorsMax = accumulators.empty() ? 900 : *std::max_element(accumulators.cbegin(), accumulators.cend());
```

Comment:
Why 900? I guess it doesn't really matter but just curious.

Reply:
I don't actually remember.  I think I tried 1 or 255 initially and had some problems.